ci-operator/step-registry/ipi/conf/ibmcloud/default/ipi-conf-ibmcloud-default-ref.yaml

```diff
@@ -0,0 +1,26 @@
+ref:
+  as: ipi-conf-ibmcloud-default
+  from_image:
+    namespace: ocp
+    name: "4.18"
+    tag: upi-installer
+  grace_period: 10m
+  commands: ipi-conf-ibmcloud-default-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+  - name: BASE_DOMAIN
+    default: "ci-ibmcloud.devcluster.openshift.com"
+    documentation: |-
+      A fully-qualified domain or subdomain name. The base domain of the cloud
+      provider is used for setting baseDomain variable of the install
+      configuration of the cluster.
+  - name: WORKERS
```

Comment:
```suggestion
  - name: COMPUTE_NODE_REPLICAS
```

Reply:
How about keeping it, since WORKERS has been ref in ./ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__ibmcloud-*-nightly-x86.yaml

Reply:
Ideally it is better to keep consistent with other platforms, but `ipi-conf-ibmcloud` already used `WORKERS`, I am okay with keeping it.